packages/vat-data/src/exo-utils.js

```diff
@@ -267,13 +274,22 @@ export const makeExoUtils = VatData => {
   harden(prepareExoClassKit);
 
   /**
-   * @template {Record<PropertyKey, CallableFunction>} M methods
-   * @param {Baggage} baggage
-   * @param {string} kindName
-   * @param {InterfaceGuard | undefined} interfaceGuard
-   * @param {M} methods
-   * @param {DefineKindOptions<{ self: M }>} [options]
-   * @returns {import('@endo/exo').Guarded<M>}
+   * @type {{
+   * <I extends TypedInterfaceGuard>(
+   * baggage: Baggage,
+   * kindName: string,
+   * interfaceGuard: I,
+   * methods: GuardedMethods<I>,
+   * options?: DefineKindOptions<{ self: GuardedMethods<I> }>)
+   * : GuardedMethods<I> & RemotableBrand<{}, GuardedMethods<I>>;
+   * <M extends Record<PropertyKey, CallableFunction>>(
+   * baggage: Baggage,
+   * kindName: string,
+   * interfaceGuard: InterfaceGuard | undefined,
+   * methods: M,
+   * options?: DefineKindOptions<{ self: M }>)
+   * : Guarded<M>;
+   * }};
```

Comment:
Please add indentation so it is possible to read this declaration. I tried and could not review it as is.

Reply:
will do! I'll opt this package into the jsdoc prettier

Reply:
That file is now reviewable, thanks. Resolving.